packages/app/ui/walletconnect-btn.tsx

```diff
@@ -32,21 +34,17 @@ const WalletConnectBtn = ({ signInWithXDR }: { signInWithXDR?: (_) => {} }) => {
   // }, [publicKey])
 
   return (
-    <Button
-      onPress={handleConnect}
-      text={
-        state === 'session-proposal'
-          ? 'Waiting for approval'
-          : state === 'session-created'
-          ? 'Connected'
-          : 'WalletConnect'
-      }
-      rightIcon={
-        <View className="px-2">
-          <WalletConnectIcon color="white" />
-        </View>
-      }
-    />
+    <Pressable onPress={handleConnect} className="btn w-full">
+      <Text className="font-bold mr-4">
+        {/*{state === 'session-proposal'*/}
```

Comment:
why is this commented out?

Reply:
Most of this button logic was already commented out, so i wanted this button to at least look how it was suppose to look